src/intelligentSuite/brands/entities/Brand.ts

```diff
@@ -28,6 +29,14 @@ export default class Brand extends BaseEntity {
     @Column({type: "simple-array", nullable: true})
     socialAccounts?: string[];
 
+    @Field(() => BrandStatus, {nullable: true})
+    @Column({
+        type: "enum",
+        nullable: true,
+        enum: BrandStatus,
+    })
+    status?: BrandStatus;
```

Comment:
```suggestion
    @Field(() => BrandStatus, {defaultValue: BrandStatus.IN_PROGRESS})
    @Column({
        type: "enum",
        enum: BrandStatus,
    })
    status: BrandStatus;
```
In case status has default value